crt_portal/cts_forms/views.py

```diff
@@ -12,10 +12,12 @@
 
 @login_required
 def IndexView(request):
+    # Sort data based on request from params, default to `created_date` of complaint
```

Comment:
Are the small code comments I added in 57e164435fa72af2fe96b2184572f165045cb509 accurate? Just wanted to double-check my understanding of what the Python code is doing.